### DISCUSSION TITLE
Ignore this

### DISCUSSION THREAD

**Discussion by ghost** [oldschoolgg/oldschoolbot#4758] [Q&A]
deleted original post

**Comment by TastyPumPum** [comment]
> @imgidedin do you still code for the bot?
> 
> if so could you look into my bankbg issue with smallbanks? #4740


Given that his last PR was over a year ago, I'm going to say he doesn't currently and wouldn't like being tagged asking to look into a silly issue. 

Your issue isn't even breaking the bot and there's currently 414 issues open so don't know why you think it needs this much attention.

**Comment by mylife212** [comment]
I agree with Pum. Gided hasn't been active in developing for the bot in a very long time, and this issue doesn't seem to warrant bringing him back from the dead to fix it.

**Comment by ghost** [comment]
> > @imgidedin do you still code for the bot?
> > 
> > if so could you look into my bankbg issue with smallbanks? #4740
> 
> 
> Given that his last PR was over a year ago, I'm going to say he doesn't currently and wouldn't like being tagged asking to look into a silly issue. 
> 
> Your issue isn't even breaking the bot and there's currently 414 issues open so don't know why you think it needs this much attention. 
> 

that's fine I shouldn't have tagged Gideon in on the issue, but it would of been more respectful to hear this from Gideon himself instead of someone else.

as for my issue, 
![SmartSelect_20230103_173513_Discord.jpg](https://user-images.githubusercontent.com/90441459/210459129-d57478bb-9d77-4dd2-8d50-c4d4e2d161d9.jpg)
![SmartSelect_20230103_173438_Discord.jpg](https://user-images.githubusercontent.com/90441459/210459115-238839f0-314a-4e30-8e57-4811a0630003.jpg)

Magna himself told me to put this on github, and a month later asked me about it again. hopefully this answers your rude remark of why you assume I think this needs so much attention, when all I did was ask a question, that was not at all involving you, and this could have gone unanswered which would have been much better then how this has actually gone down.

**Comment by ghost** [comment]
> I agree with Pum. Gided hasn't been active in developing for the bot in a very long time, and this issue doesn't seem to warrant bringing him back from the dead to fix it. 

I understand that completely now but there really isn't much I can do about that besides not let it happen again.

but tasty didn't have to be so rude about the situation.
